.github/workflows/codeql-analysis.yml

```diff
@@ -37,11 +37,11 @@ jobs:
     - name: Checkout repository
       uses: actions/checkout@v2
     
-    # Setup OpenJDK 11
+    # Setup OpenJDK
```

Comment:
I think these actions will be soon deprecated. IT would be great to update them to newer versions such as in https://github.com/strimzi/strimzi-kafka-operator/pull/8321 => but can be done also in later PR.

Reply:
yeah sure. let me take note of it so that I can create another PR for it. Or I am already going to work on the pipeline part as I will have something for the integration tests.